services/config/pmm-managed.yaml

```diff
@@ -0,0 +1,44 @@
+services:
```

Comment:
because of deadlines we have to accept this type of configs, but once we cut release branch it should be reworked.